src/main/java/com/icthh/xm/tmf/ms/activation/domain/spec/SagaTaskSpec.java

```
@@ -53,4 +55,17 @@ public void applyAsDefaultTransactionConfig(SagaTransactionSpec tx) {
             setter.accept(value);
         }
     }
+
+    @Override
+    public SagaTaskSpec clone() {
+        SagaTaskSpec cloned = null;
+        try {
+            cloned = (SagaTaskSpec) super.clone();
+        } catch (CloneNotSupportedException e) {
+            // ignore
```

bad way to ignore exceptions